targets/TARGET_Cypress/TARGET_PSOC6/lp_ticker.c

```diff
@@ -40,29 +40,38 @@
 #define LPT_INTERRUPT_PRIORITY  6
 #define LPT_INTERRUPT_SOURCE    srss_interrupt_mcwdt_1_IRQn
 #endif
-#define LPT_MCWDT_DELAY_WAIT    0                // Recommended value is 93, but then we fail function execution time test.
-
+#define LPT_MCWDT_DELAY_WAIT    93                // Recommended value is 93, but then we fail function execution time test.
+#define LPT_MCWDT_DELAY_NO_WAIT 0
 #if !defined (CY_CFG_SYSCLK_CLKLF_FREQ_HZ)
 #define CY_CFG_SYSCLK_CLKLF_FREQ_HZ    32768UL  /* Default to 32K ILO */
 #endif /* CY_CFG_SYSCLK_CLKLF_FREQ_HZ */
 
+#define MCWDT_COUNTER0_MAX_COUNT        (0xffffUL)
+#define MCWDT_COUNTER1_MAX_COUNT        (0xffffUL)
+#define MCWDT_COUNTER2_MAX_COUNT        (0xffffffffUL)
+#define MAX_MCWDT_DURATION_SEC          (35UL*60UL*60UL*1000UL)
```

Comment:
Can this value be a just a number - 126 000 000 Plus comment how it is calculated - In this way is I see that the uint32 is not overflowed

Reply:
This question hasn't been answered. @cy-vivekp Please review